### ISSUE TITLE
Curate list of ES2015 browsers

### ISSUE THREAD

**Issue by redallen** [patternfly/patternfly-react#4047]
It's important to know for some PatternFly consumers to know which browser versions PatternFly supports since they might use RHEL or other long release-cycle distros. After #4046 we should target ES2015, and with that knowledge we can curate a list of supported browsers to proudly display in our README.

**Comment by redallen** [comment]
Our transpiled code (but not necessarily its dependencies) [supports ES6.](https://www.w3schools.com/js/js_es6.asp) I don't think that information doesn't necessarily belong in a README anymore (and we added it to patternfly.org in another PR), so I'm just closing this for now.
